qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -177,10 +178,26 @@ def __json_decode__(cls, value: Dict[str, Any]) -> "BasisGateLibrary":
 
 
 class FixedFrequencyTransmon(BasisGateLibrary):
-    """A library of gates for fixed-frequency superconducting qubit architectures.
+    r"""A library of gates for fixed-frequency superconducting qubit architectures.
 
     Note that for now this library supports single-qubit gates and will be extended
     in the future.
+
+    Provided gates:
+        - x: :math:`\pi` pulse around the x-axis.
+        - sx: :math:`\pi/2` pulse around the x-axis.
+        - y: :math:`\pi` pulse around the y-axis.
+        - sy: :math:`\pi/2` pulse around the y-axis.
+
+    Parameters:
+        - duration: Duration of the pulses.
+        - σ: Standard deviation of the pulses.
+        - β: DRAG parameter of the pulses.
+        - amp: Amplitude of the pulses. If the parameters linked then ``x`` and ``y``
+          share the same parameter and ``sx`` and ``sy`` share the same parameter.
+
+    Note that the β and amp parameters may be linked between the x and y as well as between
```

Comment:
In documenting the linking here, it would be good to add that duration and sigma are always linked.

Reply:
Yes. Done here: [5314dd6](https://github.com/Qiskit/qiskit-experiments/pull/889/commits/5314dd62adc2602d0659f0b89ef1bcf7d18d9271)